9.2/run-postgresql.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash -e
+
+# Be paranoid and stricter than we should be.
+psql_identifier_regex='^[a-zA-Z_][a-zA-Z0-9_]*$'
+psql_password_regex='^[a-zA-Z0-9_~!@#$%^&*()-=<>,.?;:|]+$'
+
+function usage {
+	echo "You must specify following environment variables:"
+	echo "  \$POSTGRESQL_USER (regex: '$psql_identifier_regex')"
+	echo "  \$POSTGRESQL_PASSWORD (regex: '$psql_password_regex')"
+	echo "  \$POSTGRESQL_DATABASE (regex: '$psql_identifier_regex')"
```

Comment:
@jhadvig also, MYSQL_ROOT_PASSWORD is a special case since it doesn't have a default but not specifying it alters the behavior (remote login to root account is disabled).